system/alternatives.py

```diff
@@ -114,7 +124,16 @@ def main():
                         link = line.split()[1]
                         break
 
-    if current_path != path:
+    if not force:
+        need_to_force = False
+    elif not os.path.islink(link):
```

Comment:
Won't this result in "--force" being passed in even if the user of the task didn't pass in a "force=yes" parameter to the module?

I guess I would expect it to only pass "--force" on to update-alternatives if it was passed into the module...